internal/pkg/manager/workloadannotator/workloadannotator.go

```diff
@@ -119,6 +120,9 @@ func (r *PodReconciler) Reconcile(_ context.Context, req reconcile.Request) (rec
 	// pod is being created or updated so ensure it is linked to a seccomp/selinux profile
 	for _, profileIndex := range getSeccompProfilesFromPod(pod) {
 		profileElements := strings.Split(profileIndex, "/")
+		if len(profileElements) != pathParts {
```

Comment:
not sure if it's worth the effort, but one thing I was considering was to return a simple struct with the elements from `getSeccompProfilesFromPod` instead of `[]string` to avoid the parsing by the caller altogether.